test/js/search/gene-search-view.test.js

```diff
@@ -6,79 +6,48 @@ import * as Reach from '@reach/router'
 const fetch = require('node-fetch');
 
 import GeneSearchView from 'components/search/genes/GeneSearchView';
-import GeneResultsPanel from 'components/search/genes/GeneResultsPanel';
 import SearchPanel from 'components/search/controls/SearchPanel';
 import { PropsStudySearchProvider } from 'providers/StudySearchProvider';
 import { PropsGeneSearchProvider, GeneSearchContext, emptySearch } from 'providers/GeneSearchProvider';
 import { FeatureFlagContext } from 'providers/FeatureFlagProvider'
 import StudyResultsPanel from 'components/search/results/ResultsPanel'
+import Study from 'components/search/results/Study'
+import StudyViolinPlot from 'components/search/genes/StudyViolinPlot'
 import * as ScpAPI from 'lib/scp-api'
 
 describe('Gene search page landing', () => {
-  it('shows studies when empty', async () => {
+  it('shows study details when empty', async () => {
+    const searchState = emptySearch
+    searchState.isLoaded = true
+    searchState.results = {studies: [{name: 'foo', description: 'bar'}]}
     const wrapper = mount((
       <FeatureFlagContext.Provider value={{gene_study_filter: false}}>
         <PropsStudySearchProvider searchParams={{terms: '', facets:{}, page: 1}}>
-          <GeneSearchContext.Provider value={emptySearch}>
+          <GeneSearchContext.Provider value={searchState}>
             <GeneSearchView/>
           </GeneSearchContext.Provider>
         </PropsStudySearchProvider>
       </FeatureFlagContext.Provider>
     ))
-    expect(wrapper.find(StudyResultsPanel)).toHaveLength(1)
+    expect(wrapper.find(Study)).toHaveLength(1)
   })
+
   it('shows gene results when gene query is loaded', async () => {
-    const activeSearchState = emptySearch
-    activeSearchState.isLoaded = true
+    const searchState = emptySearch
+    searchState.isLoaded = true
+    searchState.results = {studies: [{name: 'foo', description: 'bar', gene_matches: ['agpat2']}]}
     const wrapper = mount((
       <FeatureFlagContext.Provider value={{gene_study_filter: false}}>
         <PropsStudySearchProvider searchParams={{terms: '', facets:{}, page: 1}}>
-          <GeneSearchContext.Provider  value={activeSearchState}>
+          <GeneSearchContext.Provider  value={searchState}>
             <GeneSearchView/>
           </GeneSearchContext.Provider>
         </PropsStudySearchProvider>
       </FeatureFlagContext.Provider>
     ))
-    expect(wrapper.find(GeneResultsPanel)).toHaveLength(1)
+    console.log(wrapper.debug())
```

Comment:
```suggestion
    // console.log(wrapper.debug())
```

Reply:
good comment, I'm just going to remove the line instead